src/core/task6.js

```diff
@@ -1,4 +1,4 @@
-export function removeUnnecessary(array, callback) {
+export default function removeUnnecessary(array, callback) {
```

Comment:
Fully incorrect. 

1. Combine the functions removeUnnecessary & showElements
2. Your function call should look like showFilteredArray(array, filterFunction)